.gitignore

```diff
@@ -1,8 +1,19 @@
-# Ignore config.json
-config/config.json
+# Binaries
+dist/
+
+# Editors/IDEs
+*.swp
+*.swo
+*.vim
+.vscode/
+~
+
+# Helm
+Chart.lock
+*.tgz
 
-# Ignore log files
+# App Specific
+config/config.json
 *.log
 
-# Ignore the binary
-perseus
```

Comment:
See https://github.com/mattermost/perseus/pull/6#issuecomment-1350441717

There are no issues with keeping this. Since we migrated the build outputs to `dist` we assumed that the binary in the root folder is not yet needed, and the developer would be using make dor local development.